src/games/gloomhaven/characters.js

```diff
@@ -168,7 +198,7 @@ const characterConfig = [
       'level-8': ['cleansing-force', 'inspiring-sanctity'],
       'level-9': ['angelic-ascension', 'divine-intervention'],
     },
-    nbAttackModifiers: 20,
```

Comment:
@jrmi I just fixed an issue with sunkeeper's attack modifier in the same PR that deals with sawbones addition. Could have done a separate one to separate concerns, but not sure that's worth the effort here ;-)